Cabal-syntax/src/Distribution/Compat/Newtype.hs

```diff
@@ -36,25 +37,26 @@ import Unsafe.Coerce (unsafeCoerce)
 -- Another approach would be to use @TypeFamilies@ (and possibly
 -- compute inner type using "GHC.Generics"), but we think @FunctionalDependencies@
 -- version gives cleaner type signatures.
---
+{- FOURMOLU_DISABLE -}
```

Comment:
mm, it says `disable` but it still goes on and reformats. What's up?

Reply:
I commented the CPP then had these files formatted anyway before putting the CPP back. :)

Reply:
Oh, i see... Sounds painful but necessary. Thank you!

Reply:
Yeah it was not fantastic but also only needed once :+1: